solc/jsonCompiler.cpp

```diff
@@ -250,7 +256,7 @@ string compile(StringMap const& _sources, bool _optimize, CStyleReadFileCallback
 		}
 		catch (...)
 		{
-			output["errors"].append("Unknown exception while generating compiler output.");
+			output["errors"].append("Unknown exception while generating formal method output.");
```

Comment:
This is not only the "formal" part, it is also source lists and sources and the ast json converter.